examples/manipulating_visualizing/plot_spms.py

```diff
@@ -0,0 +1,61 @@
+"""
```

Comment:
Overall, I am not sure what this example is bringing: I don't think that doing a t-test with nilearn is really something that we want to showcase, so I am not sure what the function of this example is.